packages/scripts/package.json

```diff
@@ -18,15 +18,15 @@
     "url": "https://github.com/Talend/ui.git"
   },
   "dependencies": {
-    "@babel/cli": "^7.2.0",
-    "@babel/core": "^7.2.0",
-    "@babel/plugin-proposal-class-properties": "^7.2.0",
-    "@babel/plugin-proposal-export-default-from": "^7.2.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.2.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
-    "@babel/plugin-transform-object-assign": "^7.2.0",
+    "@babel/cli": "^7.1.0",
```

Comment:
We need to downgrade babel because Storybook 4 rely on `babel-preset-react-app` that rely on babel 7.1.0 (fixed).
So if we don't have compatible versions, @talend/scripts babel version will be in @talend/scripts node_modules. Jest (babel-jest) fails to find the babel plugins :(